packages/editor-ui/src/plugins/i18n/locales/en.json

```diff
@@ -712,6 +712,7 @@
 	"executionsList.activeExecutions.none": "No active executions",
 	"executionsList.activeExecutions.header": "{running}/{cap} active executions",
 	"executionsList.activeExecutions.tooltip": "Current active executions: {running} out of {cap}. This instance is limited to {cap} concurrent production executions.",
+	"executionsList.activeExecutions.evaluationNote": "Evaluation runs appear in the list of executions but do not count towards your execution concurrency.",
```

Comment:
```suggestion
	"executionsList.activeExecutions.evaluationNote": "Evaluation runs appear in the list of executions but do not count towards your production execution concurrency.",
```

Reply:
I guess we may refine this tooltip later (text is a bit too long with this evaluation note). Also, this note may be not necessary after we mark evaluation executions with a special icon. We just want a quick UI fix explaining why numbers might not match with what user sees in the list.